dev-docker-compose.yml

```diff
@@ -54,6 +54,8 @@ services:
       - 8080:8080
     networks:
       - grafana
+    volumes:
+      - ./log/:/log/
```

Comment:
could you add `log` folder to `.gitignore` or at least check that we would not add it by accident to git?

Reply:
There is already `/log/` in `.gitignore`